metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1602,13 +1651,21 @@ public CompletableFuture<Long> beginWritingSnapshot() {
     }
 
     @Override
-    public CompletableFuture<List<AclCreateResult>> createAcls(List<AclBinding> aclBindings) {
-        return appendWriteEvent("createAcls", () -> aclControlManager.createAcls(aclBindings));
+    public CompletableFuture<List<AclCreateResult>> createAcls(
+        ControllerRequestContext context,
+        List<AclBinding> aclBindings
+    ) {
+        return appendWriteEvent("createAcls", context.deadlineNs(),
+            () -> aclControlManager.createAcls(aclBindings));
     }
 
     @Override
-    public CompletableFuture<List<AclDeleteResult>> deleteAcls(List<AclBindingFilter> filters) {
-        return appendWriteEvent("deleteAcls", () -> aclControlManager.deleteAcls(filters));
+    public CompletableFuture<List<AclDeleteResult>> deleteAcls(
+        ControllerRequestContext context,
+        List<AclBindingFilter> filters
+    ) {
```

Comment:
I do prefer it, but before we could enforce with checkstyle, we'd have to convert all of the code to use it, which isn't currently practical. At least if I understand your suggestion. I know Jason likes this style too -- I've really come around to it because having very long lines or deep indentation is very fatiguing.